content/en/docs/tutorials/mutil-cluster-service/visting-services-in-child-cluster.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Balance to Services Running across Child Clusters"
+description: "How to visit services running across child clusters"
+date: 2022-09-01
+draft: false
+weight: 1
+---
+
+For services in child clusters, we can use service name to visit them locally in parent cluster.
+
+First please make sure `kubectl clusternet` must be at or later than version v0.12.0. Please
+follow [how to upgrade clusternet plugin with krew](../../kubectl-clusternet.md).
+
+```bash
+$ kubectl clusternet version
+{
+  "gitVersion": "0.8.0",
+  "gitCommit": "42ce8b65d04d838563a411a61706a21349431b8b",
+  "buildDate": "2022-08-22T03:27:19Z",
+  "platform": "darwin/arm64"
+}
+```
+{{% alert title="Note" color="primary" %}}
+Make sure your child clusters pod CIDR can reach to each others and each cluster must use distinct pod CIDRs 
```

Comment:
If we are visiting child services from parent cluster, we only need to make sure parent cluster can visit child clusters Pod CIDR.